Parsers/ASimAuthentication/Parsers/ASimAuthenticationSentinelOne.yaml

```diff
@@ -0,0 +1,182 @@
+Parser:
+  Title: ASIM Authentication parser for SentinelOne
+  Version: '0.1'
+  LastUpdated: Jul 24 2023
+Product:
+  Name: SentinelOne
+Normalization:
+  Schema: Authentication
+  Version: '0.1.3'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/ASimAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: SentinelOne Documentation
+  Link: https://<SOneInstanceDomain>.sentinelone.net/api-doc/overview
+Description: |
+  This ASIM parser supports normalizing SentinelOne logs to the ASIM Authentication normalized schema. SentinelOne events are captured through SentinelOne data connector which ingests SentinelOne server objects such as Threats, Agents, Applications, Activities, Policies, Groups, and more events into Microsoft Sentinel through the REST API.
+ParserName: ASimAuthenticationSentinelOne
+EquivalentBuiltInParser: _ASim_Authentication_SentinelOne
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let EventResultDetailsLookup = datatable (comments_s: string, EventResultDetails: string)
+  [
+      "invalid 2FA code", "Incorrect password",
+      "IP/User mismatch", "No such user or password",
+      "invalid password", "Incorrect password",
+      "user temporarily locked 2FA attempt", "User locked",
+      "no active site", "Other"
+  ];
+  let EventFieldsLookup = datatable (
+      activityType_d: real,
+      EventType: string,
+      EventResult: string,
+      EventOriginalType: string
+  )
+  [
+      27, "Logon", "Success", "User Logged In",
+      33, "Logoff", "Success", "User Logged Out",
+      133, "Logon", "Failure", "Existing User Login Failure",
+      134, "Logon", "Failure", "Unknown User Login",
```

Comment:
"Unknown User Login" is Failed login or Success?

Reply:
As discussed, "Unknown User Login" template indicates that Unknown user tried to login but failed so we kept EventResult as Failure.